netlify.toml

```diff
@@ -1,32 +1,22 @@
 [build]
 publish = "public"
-command = "make production-build"
+command = "npm run seq -- build:preview diff:check"
 
-[build.environment]
-HUGO_VERSION = "0.126.0"
-
-[context.deploy-preview]
-command = "make preview-build"
-
-[context.branch-deploy]
-command = "make preview-build"
-
-[[redirects]]
-from = "/specification/latest"
-to = "https://theupdateframework.github.io/specification/latest/"
+[context.production]
+command = "npm run seq -- build:production diff:check"
 
 [[redirects]]
-from = "/specification/list"
-to = "https://theupdateframework.github.io/specification/"
+from = "/specification"
+to = "/specification/latest/"
 
 [[redirects]]
-from = "/specification"
-to = "https://theupdateframework.github.io/specification/latest/"
+from = "/specification/*"
+to = "https://theupdateframework.github.io/specification/:splat"
 
 [[headers]]
   for = "/*"
   [headers.values]
-    Content-Security-Policy = "default-src 'self' code.jquery.com fonts.googleapis.com fonts.gstatic.com use.fontawesome.com app.netlify.com netlify-cdp-loader.netlify.app youtube.com; frame-src youtube.com www.youtube.com"
+    Content-Security-Policy = "default-src 'self' code.jquery.com fonts.googleapis.com fonts.gstatic.com cse.google.com www.google.com use.fontawesome.com app.netlify.com netlify-cdp-loader.netlify.app youtube.com; frame-src youtube.com www.youtube.com"
```

Comment:
@JustinCappos et al.: I wanted to highlight this addition to the CSP, i.e., two google.com sites, which I added while experimenting with custom Google search for the website (#91). Please confirm that you are ok with this addition, otherwise I'll roll it back.